tests/h/notification/reply_test.py

```diff
@@ -67,27 +67,32 @@ def test_returns_none_when_parent_does_not_exist(self,
 
         assert result is None
 
-    def test_returns_none_when_parent_user_does_not_exist(self, pyramid_request, reply, user_service):
-        users = {
-            'acct:elephant@safari.net': User(username='elephant')
```

Comment:
I presume the reason for removing the username here is because `factories.User` generates one automatically. Could it cause any issues that the generated username/account ID doesn't match the one in the lookup dict?

Reply:
Not an issue, the code under test doesn't make use of it. (However the whole way the fixtures are set up in these tests, with fixed user id values etc, is complicated and fragile. Can be made better. But that's not this PR's fault.)